tao_compiler/mlir/disc/transforms/fusion_utils.h

```diff
@@ -293,6 +297,9 @@ class FusionPattern : public FusionPatternBase {
   // Create a new fusion pattern from the ops inside the lmhlo fusion op.
   explicit FusionPattern(lmhlo::FusionOp op, ShapeAnalysis* shape_analysis);
 
+  // Create a new fusion pattern with the ops inside the list.
+  explicit FusionPattern(SmallVectorImpl<Operation*>& op_list);
```

Comment:
It's actually deprecated constructor since we can not infer the placement, fusion strategy and all other related things. Do we have to introduce this again?

Reply:
Reset as private, and add a new function named createWithoutInit(list).